agents-api/tests/test_tool_calls_evaluator.py

```diff
@@ -0,0 +1,1807 @@
+    {
+        "role": "assistant",
+        "content": "Creating agent",
+        "tool_calls": [
+            {
+                "id": "call_agent_create",
+                "type": "system",
+                "function": {
+                    "name": "agent.create",
+                    "arguments": {
+                        "data": {
+                            "name": "test agent 2",
+                            "canonical_name": f"some_unique_name_{str(uuid7()).split('-')[0]}",
+                            "model": "gpt-4o-mini",
+                            "about": "test agent about 2",
+                            "metadata": {"test": "test 2"},
+                        },
+                        "developer_id": str(UUID(int=0)),
+                        "agent_id": str(uuid7()),
+                    },
+                },
+            }
+        ],
+    },
+    {
+        "role": "assistant",
+        "content": "Agent created successfully",
+    },
+])
+
+agent_update = make_acompletion_multiple_outputs(
+    lambda agent, doc, user, task, user_doc, session: [
+        {
+            "role": "assistant",
+            "content": "Updating agent",
+            "tool_calls": [
+                {
+                    "id": "call_agent_update",
+                    "type": "system",
+                    "function": {
+                        "name": "agent.update",
+                        "arguments": {
+                            "agent_id": str(agent.id),
+                            "developer_id": str(UUID(int=0)),
+                            "data": {"name": "Updated Agent", "about": "Updated description"},
+                        },
+                    },
+                }
+            ],
+        },
+        {
+            "role": "assistant",
+            "content": "Agent updated successfully",
+        },
+    ]
+)
+
+agent_delete = make_acompletion_multiple_outputs(
+    lambda agent, doc, user, task, user_doc, session: [
+        {
+            "role": "assistant",
+            "content": "Deleting agent",
+            "tool_calls": [
+                {
+                    "id": "call_agent_delete",
+                    "type": "system",
+                    "function": {
+                        "name": "agent.delete",
+                        "arguments": {
+                            "agent_id": str(agent.id),
+                            "developer_id": str(UUID(int=0)),
+                        },
+                    },
+                }
+            ],
+        },
+        {
+            "role": "assistant",
+            "content": "Agent deleted successfully",
+        },
+    ]
+)
+
+user_doc_list = make_acompletion_multiple_outputs([
+    {
+        "role": "assistant",
+        "content": "Listing user docs",
+        "tool_calls": [
+            {
+                "id": "call_user_doc_list",
+                "type": "system",
+                "function": {
+                    "name": "user.doc.list",
```

Comment:
I think this (and related) should be `user.docs.list` etc to keep things consistent with the api and sdk